### ISSUE TITLE
Using this within an application

### ISSUE THREAD

**Issue by evenstensberg** [excalidraw/excalidraw#4702]

**Comment by ad1992** [comment]
Yep you will need to use the [@excalidraw/excalidraw-next](https://npmjs.com/@excalidraw/excalidraw-next) to embed excalidraw into your application. We will be soon releasing the official version as well
